xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java

```diff
@@ -152,6 +152,29 @@ public String getSection()
         return this.section;
     }
 
+     /**
+     * @param excludeFirstHeading {@code true} to remove the first heading found inside
+     *        the document or the section, {@code false} to keep it 
+     * @since 12.3RC1 
+     */
+    @Unstable
+    @PropertyName("Exclude First Heading")
+    @PropertyDescription("Exclude the first heading from the included document or section.")
+    public void setExcludeFirstHeading(boolean excludeFirstHeading)
+    {
+        this.excludeFirstHeading = excludeFirstHeading;
+    }
+    
+    /**
+     * @return whether to exclude the first heading from the included document or section, or not.
+     * @since 12.3RC1 
+     */
+    @Unstable
+    public boolean excludeFirstHeading()
```

Comment:
boolean getters are supposed to start with "is" (or get like other getters) or they won't be recognized.

Reply:
Changing it  to `isExcludeFirstHeading()`  Thanks